common/lib/util/utils.js

```diff
@@ -129,10 +129,16 @@ var Utils = (function() {
 	 * of another constructor
 	 * See node.js util.inherits
 	 */
-	Utils.inherits = (typeof(require) === 'function' && require('util') && require('util').inherits) || function(ctor, superCtor) {
+
+	Utils.inherits = function(ctor, superCtor) {
 		ctor.super_ = superCtor;
 		ctor.prototype = Utils.prototypicalClone(superCtor.prototype, { constructor: ctor });
 	};
+	try {
+        if (typeof(require) === 'function' && require('util') && require('util').inherits) {
+        	Utils.inherits = require('util').inherits;
+		}
+    } catch (err) {  /* Do Nothing. */ }
```

Comment:
Awesome, thanks, I apologize if it seemed like I was being a pain; I love programming and to fix/debug things -- I also have to work very hard to stay inside scope sometimes...  :grinning:      

@mattheworiordan has only contracted a certain number of hours with me and stuff that occurs potentially outside of scope of my changes will impact the number of hours he has remaining.   So unfortunately unless he says yeah lets do this and Nathanael does it, I'm trying very hard not to use any of the time he has allocated for any other items without his permission.    If he want to convey to me that you have the ability to direct me also; that also would suffice as permission.  :grinning: